src/sirqueue.c

```diff
@@ -54,7 +54,7 @@ bool _sir_queue_create(sir_queue** q) {
 
     if (_sir_validptrptr(q)) {
         *q = calloc(1, sizeof(sir_queue));
-        if (!*q)
+        if (!*q) //-V595
```

Comment:
I also did not get a PVS warning here. Maybe your settings are more strict than mine, but if this is "could be null," we need to fix _sir_valid* so that the linters understand that they check against null. I think the macro->inline function is confusing them.